docs_user/modules/proc_verifying-the-image-service-adoption.adoc

```diff
@@ -2,19 +2,19 @@
 
 = Verifying the {image_service} adoption
 
-Verify that you successfully adopted your {image_service_first_ref} to the {rhos_long} {rhos_curr_ver} deployment.
+Verify that you adopted the {image_service_first_ref} to the {rhos_long} {rhos_curr_ver} deployment.
 
 .Procedure
 
-. Test the {image_service_first_ref} from the {rhos_prev_long} CLI. You can compare and make sure the configuration has been correctly applied to the {image_service} pods:
+. Test the {image_service} from the {rhos_prev_long} command line interface (CLI). You can compare and ensure that the configuration is applied to the {image_service} pods:
 +
 ----
-os-diff diff /etc/glance/glance.conf.d/02-config.conf glance_patch.yaml --frompod -p glance-api
+$ os-diff diff /etc/glance/glance.conf.d/02-config.conf glance_patch.yaml --frompod -p glance-api
 ----
 +
-If no line appears, then the configuration is correctly done.
+If no line appears, then the configuration is correct.
 
-. Inspect the resulting glance pods:
+. Inspect the resulting {image_service} pods:
 +
 ----
 GLANCE_POD=`oc get pod |grep glance-default-external-0 | cut -f 1 -d' '`
```

Comment:
Comment from hackfest: "In my test environment, I see the glance pod name is "glance-default-single-0", we can think to change the grep string" 
Link for reference: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABQ54c_1s

Reply:
Yes, if this part is generic (e.g. valid for all the backends), it makes sense to relax the grep and match more.
However, considering you don't have a single Pod, you can get the first one with the following:

```
GLANCE_POD=`oc get pod |grep glance-default | cut -f 1 -d' ' | head -n 1`
```